README.rst

```diff
@@ -37,55 +37,44 @@ Usage Example
 .. code:: python
 
     import time
+    import random
     import board
     import terminalio
     from adafruit_matrixportal.matrixportal import MatrixPortal
 
-    # You can display in 'GBP', 'EUR' or 'USD'
-    CURRENCY = "USD"
-    # Set up where we'll be fetching data from
-    DATA_SOURCE = "https://api.coindesk.com/v1/bpi/currentprice.json"
-    DATA_LOCATION = ["bpi", CURRENCY, "rate_float"]
+    # --- Display setup ---
+    matrixportal = MatrixPortal(status_neopixel=board.NEOPIXEL, debug=True)
```

Comment:
Could you remove the `debug=True`? It was intended for debugging. ;)

Reply:
As a new user, the debug=True was valuable.  I understand it's a left-over artifact, and it's one that I'm glad was there when I got started.  If you feel strongly about it, I'll remove it.

Reply:
Interesting, I didn't realize it was so helpful. I suppose we can leave it if you like.